### PR TITLE
Fix init

### PR DESCRIPTION
**Pull request by sarvaje**
## Pull request checklist

Make sure you:

- [ ] Signed the [Contributor License Agreement](https://cla.js.foundation/sonarwhal/sonarwhal)
- [ ] Followed the [commit message guidelines](https://sonarwhal.com/docs/contributor-guide/getting-started/pull-requests/#commit-messagess)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/sonarwhal/src/lib/utils/resource-loader.ts [hunk 1]
```diff
@@ -55,6 +55,34 @@ const isVersionValid = (resourcePath: string): boolean => {
  * Public
  * ------------------------------------------------------------------------------
  */
+/** Returns a list with the ids of all the core resources of the given `type`. */
```
Comment (molant) [inline packages/sonarwhal/src/lib/utils/resource-loader.ts]: Add a blank line

File: packages/sonarwhal/src/lib/utils/resource-loader.ts [hunk 2]
```diff
@@ -55,6 +55,34 @@ const isVersionValid = (resourcePath: string): boolean => {
...
+
+export const getCoreResources = (type: string): Array<string> => {
```
Comment (molant) [inline packages/sonarwhal/src/lib/utils/resource-loader.ts]: Missing description

File: packages/sonarwhal/src/lib/utils/resource-loader.ts [hunk 3]
```diff
@@ -55,6 +55,34 @@ const isVersionValid = (resourcePath: string): boolean => {
...
+
+    const resourcesFiles: Array<string> = globby.sync(`dist/src/lib/${type}s/**/*.js`, { cwd: SONARWHAL_ROOT });
+    const ids: Array<string> = resourcesFiles.reduce((list: Array<string>, resourceFile: string) => {
+
+        const resourceName: string = path.basename(resourceFile, '.js');
+
+        if (path.dirname(resourceFile).includes(resourceName)) {
+
+            list.push(resourceName);
+
```
Comment (molant) [inline packages/sonarwhal/src/lib/utils/resource-loader.ts]: Remove blank line

**Comment by sarvaje** [comment]
> It's weird that your `yarn.lock` is different when no packages have changed.

This is like `package-lock.json` always you run `yarn` it change the file...

**Comment by sarvaje** [comment]
@molant this should be ready to review/merge

**Pull request merged**